app/src/test/java/com/kickstarter/libs/utils/extensions/StringExtKtTest.kt

```diff
@@ -0,0 +1,140 @@
+package com.kickstarter.libs.utils.extensions
+
+import com.kickstarter.KSRobolectricTestCase
+import org.junit.Test
+
+class StringExtKtTest : KSRobolectricTestCase() {
+
+    @Test
+    fun isEmail_whenGivenEmail_shouldReturnTrue() {
+        assertTrue(VALID_EMAIL.isEmail())
+    }
+
+    @Test
+    fun isEmail_whenNotGivenEmail_shouldReturnFalse() {
+        assertFalse(INVALID_EMAIL.isEmail())
+    }
+
+    @Test
+    fun isEmpty_whenGivenEmptyString_shouldReturnTrue() {
+        assertTrue(EMPTY_STRING.isEmpty())
```

Comment:
No need for this test, isEmpty is an String function both in Java and kotlin  java -> https://www.javatpoint.com/java-string-isempty, Kotlin -> https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/is-empty.html

Same with all tests for the functionallities already existing in the languages

Reply:
This is testing the isEmpty extension function in the StringsExt file, not the isEmpty java/kotlin function.